test/regression/RosenbrockChapman/regression_test_p_force_policy.hpp

```diff
@@ -39,7 +39,7 @@ void testRateConstants(OdeSolverPolicy& solver)
     EXPECT_EQ(state.rate_constants_[i].size(), fixed_state.rate_constants_[0].size());
     for (size_t j{}; j < state.rate_constants_[i].size(); ++j)
     {
-      EXPECT_EQ(state.rate_constants_[i][j], fixed_state.rate_constants_[0][j]);
+      EXPECT_DOUBLE_EQ(state.rate_constants_[i][j], fixed_state.rate_constants_[0][j]);
```

Comment:
If it fails with Intel compiler only, maybe it is related to the extend 80-bit precision used by the Intel compiler (https://en.wikipedia.org/wiki/Extended_precision#:~:text=The%20x86%20extended%20precision%20format,floating%2Dpoint%20arithmetic%20in%20hardware.)? No change is needed and I just want to provide additional information.

Reply:
fixed by setting compiler flags